twitter/src/test/scala/org/atnos/eff/addon/twitter/AsyncTwitterFutureInterpreterSpec.scala

```diff
@@ -0,0 +1,184 @@
+package org.atnos.eff.addon.twitter
+
+import scala.collection.mutable.ListBuffer
+import scala.concurrent._
+import scala.concurrent.duration._
+import cats.implicits._
+import com.twitter
+import com.twitter.util.{Await, Future}
+import org.atnos.eff._
+import org.atnos.eff.all._
+import org.atnos.eff.syntax.all._
+import org.scalacheck._
+import org.specs2._
+import org.specs2.concurrent.ExecutionEnv
+import org.specs2.matcher.ThrownExpectations
+
+class AsyncTwitterFutureInterpreterSpec(implicit ee: ExecutionEnv) extends Specification with ScalaCheck with ThrownExpectations { def is = s2"""
+
+ Async effects can be implemented with an AsyncFuture service                      $e1
+ Async effects can be attempted                                                    $e2
+ Async effects can be executed concurrently                                        $e3
+ Async effects are stacksafe with asyncFork                                        $e4
+ Async effects are stacksafe with asyncDelay                                       $e5
+ Async effects can trampoline a Task                                               $e6
+ An Async effect can be created from Either                                        $e7
+ An Async forked computation can be timed out                                      $e8
+ An Async forked computation which throws an exception can be executed concurently $e14
+
+ Simple Async calls can be memoized                 $e9
+ Attempted Async calls can be memoized              $e10
+ Simple Async calls with timeout can be memoized    $e11
+ Attempted Async calls with timeout can be memoized $e12
+
+ Async calls can be memoized with a memo effect $e10
```

Comment:
That's what I mean, there is `e10` twice, here and on line 30 and I think that here should be `e13`.